### ISSUE TITLE
Can't import to Android Studio

### ISSUE THREAD

**Issue by borjaruiz** [andraskindler/quickscroll#12]
How to import to android studio?

I tried compile 'com.andraskindler.quickscroll:quickscroll:0.9.8' but I can't import any library class to my Activities/fragments.

I also tried to importing it as module but Android Studio shows that it doesn't find Gradle.

**Comment by bilthon** [comment]
I recognize this must be quite late, but I actually ported this to work with the gradle build system and created a [pull request](https://github.com/andraskindler/quickscroll/pull/10). It was never accepted however. I think this project is not being mantained.

But if you pull from my fork it should work. Just add this to your build.gradle:
```
compile 'com.github.bilthon:quickscroll:v1.0.0'
```

**Comment by lorenzoPrimi** [comment]
@bilthon doesn't work

> Failed to resolve: com.github.bilthon:quickscroll:v1.0.0

**Comment by bilthon** [comment]
You're right, I never published it to maven central, so it won't work. I don't know what I was thinking of at the time. But if you check my [forked repo](https://github.com/bilthon/quickscroll) you'll actually have a better idea of how to include it using gradle.
